io_directory_example/settings/SL2SJ0219-0829_config.yml

```diff
@@ -0,0 +1,47 @@
+---
+system_name: 'SL2SJ0219-0829'
```

Comment:
Change the SL2S naming for this, and use a generic name, e.g., "lens_system3". Change the data and psf files accordingly.

Reply:
If you are adding a third example system for checking multi-filter modeling, why did you add a second filter for "lens_system1"? Was that also necessary? If not, please remove unused files.